src/levelmeter.cpp

```diff
@@ -60,7 +59,10 @@ CLevelMeter::CLevelMeter ( QWidget* parent ) :
     // initialize bar meter
     pBarMeter = new QProgressBar();
     pBarMeter->setOrientation ( Qt::Vertical );
-    pBarMeter->setRange ( 0, 100 * NUM_STEPS_LED_BAR ); // use factor 100 to reduce quantization (bar is continuous)
+    pBarMeter->setRange (
```

Comment:
Yes...  It needs to prefer to split a trailing comment in order to get a line to fit (and then put a blank line (if it's a full statement like this) and the comment before where it got it from).

Reply:
looks fine with `ColumnLimit: 120`